sections/futureWork.tex

```diff
@@ -0,0 +1,8 @@
+\section{Future Work}\label{sec:future_work}
+Due to the promising results provided by the modified pachinko model, investigating this model further might be beneficial to incorporating meta-data into topic models.
+However, testing these model on multiple datasets, needed to be accounted for, since the generalizability of these models is not explored within this paper.
+using word embedding to further improve the performance of models can also be viewed as the next step for this project, since a wide number of papers are using this technique to improve topic modeling.

```

Comment:
```suggestion
Using word embedding to further improve the performance of models can also be viewed as the next step for this project, since a wide number of papers are using this technique to improve topic modeling.

```